ci/tasks/light-aws/cleanup-ami.yml

```
@@ -0,0 +1,19 @@
+---
+platform: linux
+
+
+inputs:
+- name: bosh-stemcells-ci
+
+run:
+  path: bosh-stemcells-ci/ci/tasks/light-aws/cleanup-ami.sh
+params:
+  ami_region:                 "eu-central-1" # AWS default region
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)